configs/interaction/environment/audio_dataset.yaml

```
@@ -0,0 +1,15 @@
+_target_: ami.interactions.environments.dummy_environment.DummyEnvironment
+
+observation_generator:
+  _target_: ami.interactions.environments.audio_observation_generator.AudioFilesObservationGenerator
+  audio_files: ??? # Please specify dataset dir.
+  chunk_size: ${shared.audio_sample_size}
+  stride: ${shared.audio_sample_size}
```

strideは0.1秒ごとにすると実動と同じになる。10fpsで動作するからね。
```suggestion
  stride: ${python.eval":int(${shared.audio_sample_rate} * 0.1)"}
```